### ISSUE TITLE
ict files for qrc

### ISSUE THREAD

**Issue by roberthuang0306** [The-OpenROAD-Project/asap7#5]
Is it possible to provide the ICT file from which the qrcTechFile was generated?

**Comment by george-goudroumanis** [comment]
Could you please upload the ict file into the qrc directory?

**Comment by maliberty** [comment]
I don't think so as it is a proprietary format.  You could contact ASU and see if they will share it.
